CrustTests/Company.swift

```diff
@@ -14,6 +14,35 @@ class Company {
 
 extension Company: AnyMappable { }
 
+enum CompanyCodingKey: Keypath {
+    case uuid
+    case employees(Set<EmployeeCodingKey>)
+    case founder
+    case name
+    case foundingDate
+    case pendingLawsuits
+    
+    var keyPath: String {
+        switch self {
+        case .uuid:             return "data.uuid"
+        case .employees(_):     return "employees"
+        case .founder:          return "founder"
+        case .name:             return "name"
+        case .foundingDate:     return "data.founding_date"
+        case .pendingLawsuits:  return "data.lawsuits.pending"
```

Comment:
Is there a world in which this keypath is different based on the endpoint or something?